.github/workflows/security.yml

```
@@ -0,0 +1,90 @@
+name: Security
+
+env:
+  SECURITY_SEVERITY_THRESHOLD: high
+
+  # Ignore `project.json` as snyk thinks it is a dot net project
+  SECURITY_BASE_IGNORE_LIST: ".nx,project.json"
+
+  # Ignore packages that have issues when run with `synk test --all-projects`
+  # NOTE: Ignore list can't be paths
+  SECURITY_PACKAGE_IGNORE_LIST: "ag-grid-angular,ag-grid-enterprise,ag-grid-vue3,ag-grid-generate-code-reference-files,ag-grid-generate-example-files,module-size,module-size-angular"
+
+  # Paths of `SECURITY_PACKAGE_IGNORE_LIST`.
+  # Paths are required to run snyk tests individually
+  SECURITY_PACKAGE_IGNORE_LIST_PATHS: "packages/ag-grid-angular/package.json,packages/ag-grid-angular/projects/ag-grid-angular/package.json,packages/ag-grid-enterprise/package.json,packages/ag-grid-vue3/package.json,plugins/ag-grid-generate-code-reference-files/package.json,plugins/ag-grid-generate-example-files/package.json,testing/module-size/package.json,testing/module-size-angular/package.json"
+
+on:
+    issue_comment:
+        types: [created]
+    workflow_dispatch:
+    schedule:
+      - cron: '0 0 * * *' # Run daily at midnight UTC
+    push:
+      branches: ["AG-3390-snyk-gha"]
+    pull_request:
+      branches: ["AG-3390-snyk-gha"]
+
+jobs:
+  security-all-projects:
+    runs-on: ubuntu-latest
+    if: github.event_name == 'workflow_dispatch' || (github.event_name == 'issue_comment' && github.event.comment.body == '/security') || github.event_name == 'schedule' || github.event_name == 'push' || github.event_name == 'pull_request'
```

To be removed when review is passed

```suggestion
    if: github.event_name == 'workflow_dispatch' || (github.event_name == 'issue_comment' && github.event.comment.body == '/security') || github.event_name == 'schedule'
```